Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -2654,15 +2731,23 @@ def splunk_submit_event_hec(
     if request_channel:
         headers['X-Splunk-Request-Channel'] = request_channel
 
+    data = ''
+    if entry_id or batch_event_data:
+        data = events
+    else:
+        data = json.dumps(events)
+
+    demisto.debug(f'Splunk sending {data=}')
```

Comment:
I would consider just adding number of events as the data is visible using the CLI (or with the file provided if used)